var/spack/repos/builtin/packages/gatk/package.py

```diff
@@ -39,17 +39,26 @@ class Gatk(Package):
 
     version('3.8-0', '0581308d2a25f10d11d3dfd0d6e4d28e', extension='tar.gz',
             url="https://software.broadinstitute.org/gatk/download/auth?package=GATK")
+    version('4.0.4.0', '083d655883fb251e837eb2458141fc2b',
+            url="https://github.com/broadinstitute/gatk/releases/download/4.0.4.0/gatk-4.0.4.0.zip")
```

Comment:
Can you sort versions from newest to oldest?

Reply:
Sure. I'll fix it.